src/Compilers/CSharp/Test/Semantic/SourceGeneration/StateTableTests.cs

```diff
@@ -1104,6 +1104,76 @@ public void Node_Table_When_Previous_Was_Larger_Multi()
             AssertTableEntries(table.AsCached(), ImmutableArray.Create(("class1", EntryState.Cached, 0), ("class1.1", EntryState.Cached, 1)));
         }
 
+        [Fact, WorkItem(66451, "https://github.com/dotnet/roslyn/issues/67123")]
```

Comment:
The numbers don't match. Consider using `WorkItem` overload with only the URL string. (Below as well.)